src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
> I don't think neither of these are correct, but I am taking you just meant managed by rnpm

Yeah :)

> This is technically speaking OK because most Android apps have strings.xml, bools.xml, arrays.xml and so on - however, that means if in the future, we decide to add e.g. <bool /> - we will have to either put it in strings.xml (breaks convention) or to put it in bools.xml (makes one more file we have to load, parse and extract rnpm variables from). Having a single file just makes it easier to manage them all in one place, however at this point I understand there might be no obvious way for having it.

I totally agree with this. I think at some point (maybe even very soon) we'll have "types" for our questions and we'll be able to group answers by type (bools, strings, numbers etc) if that's what Google recommends for Android development.

> I am not against it as (if you remember), I suggested it initially in one of the issues. rnpm.xml was just an idea I had in my mind as a drop-in replacement for moduleConfig (which in fact should be named rnpmConfig to be more explicit about its source).

Yeah, that's why I used it, actually :) I think if the approach with splitting vars by type wouldn't work, I'd say we need to go with rnpm.xml approach, but I want to give it a chance first.